tests/basic/storage_pool_test.go

```diff
@@ -1408,16 +1415,52 @@ func addCloudDrive(stNode node.Node, poolID int32) error {
 	deviceSpec := driveSpecs[0]
 	deviceSpecParams := strings.Split(deviceSpec, ",")
 	var specSize uint64
+	var driveSize string
+
+	if poolID != -1 {
+		systemOpts := node.SystemctlOpts{
+			ConnectionOpts: node.ConnectionOpts{
+				Timeout:         2 * time.Minute,
+				TimeBeforeRetry: defaultRetryInterval,
+			},
+			Action: "start",
+		}
+		drivesMap, err := Inst().N.GetBlockDrives(stNode, systemOpts)
+		log.FailOnError(err, "error getting block drives from node %s", stNode.Name)
```

Comment:
Same here, lets `return err` instead of `log.FailOnError()` for consistency

Reply:
fixed